calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.0-release-notes.mdx

```diff
@@ -0,0 +1,7 @@
+30 May 2023
+
+### What's new {#whats-new-3.17.0}
+
+:::note
+Due to a tagging error on our part, 3.17.0 was not released. The first release of 3.17 is 3.17.1.
```

Comment:
> Due to a tagging error on our part, 3.17.0 was not released.

Is this what we want to tell the user? What really happened is that we made v3.17.0, followed by v3.17.1 with some fixes. Both are early preview.

Reply:
If we do mention the fact that there's no .0 release, we can simply mention the fact without going into details.

Reply:
It has been removed in both places.